Source/EventFlow.Autofac.Tests/EventFlow.Autofac.Tests.csproj

```diff
@@ -0,0 +1,116 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{EDCD8854-6224-4329-87C2-9ADD7D153070}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>EventFlow.Autofac.Tests</RootNamespace>
+    <AssemblyName>EventFlow.Autofac.Tests</AssemblyName>
+    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
```

Comment:
@JC008 Please set the framework to 4.5.1. Ideally multiple frameworks would be included in the NuGet, but I haven't create a setup for that... yet.